example/schehduling/federateddeployment-rsp.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: scheduling.federation.k8s.io/v1alpha1
```

Comment:
Note, there's a typo in the filepath here.

Reply:
Done